### PR TITLE
Smokebomb changes.

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

This is the most critical PR of 2k19.
Changes smoke grenade description.
Adds extremely useful documentation.
Does some general sanity changes, but who gives a fuck about that?

## Why It's Good For The Game

It's Hip.

## Changelog
:cl:
tweak: Smoke grenades are now double dank.
imageadd: Inhands for smokebombs.
/:cl:

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
needs a sound effect that says "smoke bomb ev'ry day"

File: code/game/objects/items/grenades/smokebomb.dm [hunk 1]
```diff
@@ -1,30 +1,32 @@
+/**
+  *This is smoke bomb, mezum koman. It is a grenade subtype. All craftmanship is of the highest quality.
+  *It menaces with spikes of iron. On it is a depiction of an assistant.
+  *The assistant is bleeding. The assistant has a painful expression. The assistant is dead.
+  */
 /obj/item/grenade/smokebomb
 	name = "smoke grenade"
-	desc = "The word 'Dank' is scribbled on it in crayon."
+	desc = "Real bruh moment if you ever see this. Probably tell a c*der or something."
```
Comment (actioninja) [inline code/game/objects/items/grenades/smokebomb.dm]: Smoke grenades are obtainable in game, real bruh moment right here.

**Review by actioninja — CHANGES_REQUESTED** [review]

File: code/game/objects/items/grenades/smokebomb.dm (hunk 1 above)
Comment (actioninja) [reply code/game/objects/items/grenades/smokebomb.dm]: the real bruh moment was within me all along, I just realized what you did.

**Review by actioninja — APPROVED** [review]
I'm just an idiot

**Comment by Bawhoppen** [comment]
I'm still not sure why you made this change but 👍

**Pull request merged**